# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1279516965 25200
#      Sun Jul 18 22:22:45 2010 -0700
# Node ID eaf7c8ab3b0c7bd7e5eb0d62a9533fe2895b6894
# Parent  0306e572c686946231a9370de78405c2a3776a20
Replace 8 spaces with tabs in ChangeLogs

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,91 +1,84 @@
 2010-07-18  Rik <octave@nomad.inbox5.com>
 
-        * interpreter/doccheck/spellcheck,
+	* interpreter/doccheck/spellcheck,
 	interpreter/doccheck/spellcheck/add_to_aspell_dict:
 	Use case insensitive sort in spellcheck scripts.
 
 2010-07-18  Rik <octave@nomad.inbox5.com>
 
-        * interpreter/doccheck/aspell-octave.en.pws: Add Octave-specific words
+	* interpreter/doccheck/aspell-octave.en.pws: Add Octave-specific words
 	from stats.texi to private Aspell dictionary.
 
 2010-07-17  Rik <octave@nomad.inbox5.com>
 
-        * interpreter/doccheck: New directory for spelling/grammar scripts.
-        * interpreter/doccheck/README: Instructions for using scripts.
-        * interpreter/doccheck/spellcheck: Script to spellcheck a Texinfo file.
-        * interpreter/doccheck/aspell.conf: GNU Aspell configuration file for
-        Octave documentation.
-        * interpreter/doccheck/aspell-octave.en.pws: Private Aspell dictionary.
-        * interpreter/doccheck/add_to_aspell_dict: Script to add new 
-        Octave-specific words to private Aspell dictionary.
-        * interpreter/octave.texi: New @nospell macro which forces Aspell to
-          ignore the word marked by the macro.
-        * interpreter/mk_doc_cache.m: Skip new @nospell macro when 
-          building doc_cache.
-        * interpreter/arith.txi, interpreter/audio.txi, 
-          interpreter/bugs.txi, interpreter/contrib.txi, 
-          interpreter/debug.txi, interpreter/diagperm.txi,
-          interpreter/dynamic.txi, interpreter/geometry.txi,
-          interpreter/matrix.txi, interpreter/numbers.txi, 
-          interpreter/oop.txi, interpreter/package.txi,
-          interpreter/sparse.txi, interpreter/tips.txi: 
-          Spellcheck documentation.
-
-          DLD-FUNCTIONS/balance.cc, DLD-FUNCTIONS/ccolamd.cc,
-          DLD-FUNCTIONS/cellfun.cc, DLD-FUNCTIONS/dasrt.cc,
-          DLD-FUNCTIONS/dmperm.cc, DLD-FUNCTIONS/eigs.cc, DLD-FUNCTIONS/qz.cc,
-          DLD-FUNCTIONS/rand.cc, DLD-FUNCTIONS/sparse.cc, 
-          DLD-FUNCTIONS/sqrtm.cc, DLD-FUNCTIONS/symrcm.cc, dirfns.cc,
-          file-io.cc, syscalls.cc
+	* interpreter/doccheck: New directory for spelling/grammar scripts.
+	* interpreter/doccheck/README: Instructions for using scripts.
+	* interpreter/doccheck/spellcheck: Script to spellcheck a Texinfo file.
+	* interpreter/doccheck/aspell.conf: GNU Aspell configuration file for
+	Octave documentation.
+	* interpreter/doccheck/aspell-octave.en.pws: Private Aspell dictionary.
+	* interpreter/doccheck/add_to_aspell_dict: Script to add new 
+	Octave-specific words to private Aspell dictionary.
+	* interpreter/octave.texi: New @nospell macro which forces Aspell to
+	  ignore the word marked by the macro.
+	* interpreter/mk_doc_cache.m: Skip new @nospell macro when 
+	  building doc_cache.
+	* interpreter/arith.txi, interpreter/audio.txi, 
+	  interpreter/bugs.txi, interpreter/contrib.txi, 
+	  interpreter/debug.txi, interpreter/diagperm.txi,
+	  interpreter/dynamic.txi, interpreter/geometry.txi,
+	  interpreter/matrix.txi, interpreter/numbers.txi, 
+	  interpreter/oop.txi, interpreter/package.txi,
+	  interpreter/sparse.txi, interpreter/tips.txi: 
+	  Spellcheck documentation.
 
 2010-07-01  Rik <octave@nomad.inbox5.com>
 
-        * refcard/refcard.tex: Fix italic macro typo for stem.
+	* refcard/refcard.tex: Fix italic macro typo for stem.
 
 2010-06-28  Rik <octave@nomad.inbox5.com>
 
-        * refcard/refcard.tex: Fix typos in refcard (bug #30260).
+	* refcard/refcard.tex: Fix typos in refcard (bug #30260).
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
-        * ChangeLog: Correctly break lines > 80 characters.
+	* ChangeLog: Correctly break lines > 80 characters.
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
-        * interpreter/intro.txi: Clarify wording.  (bug #30165).
+	* interpreter/intro.txi: Clarify wording.  (bug #30165).
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
-        * interpreter/set.txi: Remove reference to deprecated function.
+	* interpreter/set.txi: Remove reference to deprecated function.
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
-        * interpreter/expr.txi, interpreter/func.txi, interpreter/install.txi,
-        interpreter/plot.txi, interpreter/sparse.txi: Periodic grammar check
-        of documentation to ensure a common format.
+	* interpreter/expr.txi, interpreter/func.txi, interpreter/install.txi,
+	interpreter/plot.txi, interpreter/sparse.txi: Periodic grammar check
+	of documentation to ensure a common format.
 
 2010-05-27  Rik <octave@nomad.inbox5.com>
 
-        * interpreter/arith.txi, interpreter/diffeq.txi, interpreter/intro.txi,
-        interpreter/nonlin.txi, interpreter/poly.txi, interpreter/sparse.txi:
-        Replace @ifinfo with @ifnottex.
+	* interpreter/arith.txi, interpreter/diffeq.txi, interpreter/intro.txi,
+	interpreter/nonlin.txi, interpreter/poly.txi, interpreter/sparse.txi:
+	Replace @ifinfo with @ifnottex.
 
 2010-05-07  John W. Eaton  <jwe@octave.org>
 
 	* basics.txi: Update list of internal variables for --traditional.
 
 2010-05-02  Rik <octave@nomad.inbox5.com>
 
-        * interpreter/contrib.txi, interpreter/install.txi,
-        interpreter/plot.txi, interpreter/sparse.txi,
-        interpreter/octave.texi: Remove tabs causing problems with
-        pdf formatting.
+	* interpreter/contrib.txi, interpreter/install.txi,
+	interpreter/plot.txi, interpreter/sparse.txi,
+	interpreter/octave.texi: Remove tabs causing problems with
+	pdf formatting.
 
 2010-04-14  Shai Ayal  <shaiay@users.sourceforge.net>
 
 	* interpreter/plot.txi: Document keypress and mouse callback
 	properties
 
 2010-04-07  John W. Eaton  <jwe@octave.org>
 
@@ -142,17 +135,17 @@ 2010-01-17  Thorsten Meyer  <thorsten.me
 2010-01-09  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/munge-texi.cc: Avoid C-style casts.
 	(main): Call usage if fewer than two arguments.
 
 2009-12-20  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/geometryimages.m: Correctly set postscript output when
-        QHULL library does not exist and sombrero image is printed.
+	QHULL library does not exist and sombrero image is printed.
 
 2009-12-19  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am, faq/Makefile.am, interpreter/Makefile.am,
 	liboctave/Makefile.am, refcard/Makefile.am, interpreter/images.awk:
 	Eliminate TOPDIR variable in favor of built-in automake variables of
 	top_builddir and top_srcdir.
 
@@ -163,37 +156,37 @@ 2009-12-17  Rik  <octave@nomad.inbox5.co
 2009-12-11  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: remove install-local target and use automake
 	syntax to install doc-cache file.
 
 2009-12-10  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: Replace distdir-local with dist-hook rule as
-        it is better supported by automake.
+	it is better supported by automake.
 
 2009-12-10  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: new distdir-local rule to check for presence
-        of necessary libraries before packaging distribution
+	of necessary libraries before packaging distribution
 
 2009-12-08  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: Distribute all generated documentation
-        (.pdf, .ps, .html, .dvi) to remove build requirement for TexInfo
+	(.pdf, .ps, .html, .dvi) to remove build requirement for TexInfo
 
 2009-12-07  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am, faq/Makefile.am, liboctave/Makefile.am:
-        Document use of NOTPARALLEL directive.
+	Document use of NOTPARALLEL directive.
 
 2009-12-07  Rik <octave@nomad.inbox5.com>
 
 	* refcard/Makefile.am: Add CLEANFILES and MAINTAINERCLEANFILES objects
-        to properly clean directory.
+	to properly clean directory.
 
 2009-12-05  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: Set dependency of *.texi files on
 	munge-texi source code rather than executable.
 
 2009-12-04  Rik <octave@nomad.inbox5.com>
 
@@ -217,17 +210,17 @@ 2009-12-04  Rik <octave@nomad.inbox5.com
 	* interpreter/geometryimages.m, interpreter/interpimages.m,
 	  interpreter/sparseimages.m, interpreter/plotimages.m:
 	  Set default paperorientation to landscape to stop warning message.
 
 2009-12-04  Ben Abbott <bpabbott@mac.com>
 
 	* interpreter/geometryimages.m, interpreter/interpimages.m,
 	  interpreter/sparseimages.m: Set default papersize to the desired
-          imagesize.
+	  imagesize.
 
 2009-12-04  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/images.awk: Emit rules to copy PNG files to
 	octave.html directory once it is created.
 	* interpreter/Makefile.am (all-local): Depend on $(HTMLDIR_IMAGES).
 	(IMAGES): Include $(HTMLDIR_IMAGES) in the list.
 
@@ -295,17 +288,17 @@ 2009-12-03  John W. Eaton  <jwe@octave.o
 2009-12-02  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: Correct Makefile.am to allow VPATH builds of
 	documentation
 
 2009-12-01  Rik <octave@nomad.inbox5.com>
 
 	* faq/Makefile.am, doc/interpreter/bugs.txi,
-        doc/interpreter/install.txi: Eliminate false dependency on conf.texi.
+	doc/interpreter/install.txi: Eliminate false dependency on conf.texi.
 
 2009-11-30  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am:
 	Overhaul Makefile to correctly generate *.texi files and to stop their
 	distribution when running 'make dist'
 
 2009-11-30  John W. Eaton  <jwe@octave.org>
@@ -337,40 +330,40 @@ 2009-11-27  David Bateman  <dbateman@fre
 	* interpreter/install.txi: Document that curl libraries impact ftp
 	objects.
 	* interpreter/system.txi: Document the ftp objects
 
 2009-11-25  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am:
 	Simplify doc-cache handling in Makefile.am to use only
-        install-data-local target
+	install-data-local target
 
 2009-11-24  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am:
 	Correct Makefile so that it halts on error when unable to build
-        doc-cache.
+	doc-cache.
 
 2009-11-23  Rik <octave@nomad.inbox5.com>
 
 	* faq/Makefile.am, interpreter/Makefile.am, liboctave/Makefile.am:
 	Stop distribution of conf.texi by autotools.  conf.texi must be re-made
 	from conf.texi.in during configure step
 
 2009-11-23  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Add . to SUBDIRS to force automake to compile this
 	directory, and make conf.texi, before recursively compiling SUBDIRS
-        which depend on conf.texi
+	which depend on conf.texi
 
 2009-11-11  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Add DISTCLEANFILES macro so that generated conf.texi can
-        be cleaned by automake.
+	be cleaned by automake.
 
 2009-11-10  John W. Eaton  <jwe@octave.org>
 
 	* Makefile.am, faq/Makefile.am, interpreter/Makefile.am,
 	liboctave/Makefile.am, refcard/Makefile.am,
 	interpreter/config-images.sh, interpreter/images,
 	interpreter/images.awk: New files.
 	* Makefile.in, faq/Makefile.in, interpreter/Makefile.in,
@@ -378,17 +371,17 @@ 2009-11-10  John W. Eaton  <jwe@octave.o
 	* interpreter/munge-texi.cc (main, usage): Eliminate unnecessary
 	-d option.
 	* liboctave/liboctave.texi: Surround @direntry in appropriate magic.
 	* faq/OctaveFAQ.texi: Rename from faq/Octave-FAQ.texi.
 
 2009-10-23  Rik <octave@nomad.inbox5.com>
 
 	* various: Periodic grammar check of Octave documentation files to
-        ensure a standardized format.
+	ensure a standardized format.
 
 2009-10-20  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.in: Clean Makefile.in and remove a few
 	non-functional targets
 
 2009-10-13  Rik <octave@nomad.inbox5.com>
 
@@ -468,19 +461,19 @@ 2009-06-07  Rik  <rdrider0-list@yahoo.co
 
 2009-06-07  Thorsten Meyer  <thorsten.meyier@gmx.de>
 
 	* interpreter/container.txi: Udate documentation of container types
 
 2009-06-03  Rik  <rdrider0-list@yahoo.com>
 
 	* interpreter/plotimages.m: Correct errorbar plot for docs.  Make hist
-        plot reprodroducible for docs.
+	plot reprodroducible for docs.
 	* interpreter/interpimages.m: Correct typo in legend of interpft plot
-        for docs.
+	for docs.
 
 2009-06-03  Jaroslav Hajek  <highegg@gmail.com>
 
 	* Makefile.in: Omit conf.texi in DISTFILES.
 
 2009-06-02  Jaroslav Hajek  <highegg@gmail.com>
 
 	* interpreter/oop.txi: Update docs of polynomial class, mention
@@ -501,28 +494,28 @@ 2009-05-25  Rik  <rdrider0-list@yahoo.co
 
 2009-05-24  Robert T. Short  <octave@phaselockedsystems.com>
 
 	* interpreter/oop.txi: Added inheritance documentatin
 
 2009-05-21  Rik  <rdrider0-list@yahoo.com>
 
 	* interpreter/Makefile.in, texmf.cnf: Change texi2pdf call to include
-        local TeX configuration file which increases save_history variable.
+	local TeX configuration file which increases save_history variable.
 
 2009-05-17  Rik  <rdrider0-list@yahoo.com>
 
 	* interpreter/tips.txi: Change recommendation to use @tex rather than
 	@iftex construction.
 
 2009-05-17  Rik  <rdrider0-list@yahoo.com>
 
 	* interpreter/*.txi: Simplify TeXinfo files by eliminating redundant
-        @iftex followed by @tex construction.  Also, spellchecked all .txi
-        and .texi files.
+	@iftex followed by @tex construction.  Also, spellchecked all .txi
+	and .texi files.
 
 2009-05-14  Rik  <rdrider0-list@yahoo.com>
 
 	* interpreter/arith.txi: Update section 17.8 (Mathematical Consants)
 	of arith.txi.
 
 2009-05-01  Rik  <rdrider0-list@yahoo.com>
 
diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -36,17 +36,17 @@ 2010-06-17  Jaroslav Hajek  <highegg@gma
 
 	* MatrixType.cc (matrix_real_probe): Use OCTAVE_LOCAL_BUFFER for
 	temporary array.
 	(matrix_complex_probe): Parametrize by real type. Use
 	OCTAVE_LOCAL_BUFFER for temporary array.
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
-        * DASSL-opts.in: Periodic grammar check on documentation.
+	* DASSL-opts.in: Periodic grammar check on documentation.
 
 2010-06-16  David Bateman  <dbateman@free.fr>
 
 	* oct-rand.cc (void octave_rand:do_reset (void),
 	void octave_rand:do_reset (const std::string&)): New functions
 	* oct-rand.h (void octave_rand:do_reset (void),
 	void octave_rand:do_reset (const std::string&)): Declare them
 	(void octave_rand:reset (void),
@@ -213,21 +213,21 @@ 2010-04-18  Jaroslav Hajek  <highegg@gma
 	* Array.h: Declare it.
 	* Sparse.cc (Sparse<T>::cat): New method.
 	* Sparse.h: Declare it.
 
 2010-04-16  David Bateman  <dbateman@free.fr>
 
 	* Sparse.cc (template <class T> Sparse<T>::Sparse (const Array<T>&,
 	const idx_vector&, const idx_vector&, octave_idx_type,
-        octave_idx_type, bool, octave_idx_type)): Add argument defining the
+	octave_idx_type, bool, octave_idx_type)): Add argument defining the
 	minimum storage to allocate for the sparse matrix.
 	* Sparse.h (template <class T> Sparse (const Array<T>&,
 	const idx_vector&, const idx_vector&, octave_idx_type,
-        octave_idx_type, bool, octave_idx_type)): ditto.
+	octave_idx_type, bool, octave_idx_type)): ditto.
 	* MSparse.h : ditto
 	* CSparse.h : ditto
 	* dSparse.h : ditto
 	* boolSparse.h : ditto
 
 2010-04-14  Jaroslav Hajek  <highegg@gmail.com>
 
 	* Sparse.cc: Update failing tests.
diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -137,30 +137,30 @@ 2010-07-18  Rik <octave@nomad.inbox5.com
 	scripts/testfun/example.m, scripts/testfun/fail.m,
 	scripts/testfun/speed.m, scripts/testfun/test.m,
 	scripts/time/calendar.m, scripts/time/date.m, scripts/time/datenum.m,
 	scripts/time/datetick.m, scripts/time/datevec.m:
 	Grammarcheck documentation of m-files.
 
 2010-07-17  Rik <octave@nomad.inbox5.com>
 
-        * audio/lin2mu.m, general/common_size.m, general/interp1.m, 
-        general/interp2.m, general/interpn.m, general/quadl.m,
-        geometry/dsearchn.m, geometry/voronoi.m, geometry/voronoin.m,
-        image/hsv2rgb.m, image/imread.m, image/rgb2hsv.m,
-        linear-algebra/commutation_matrix.m, linear-algebra/condest.m,
-        linear-algebra/expm.m, linear-algebra/krylov.m,
-        linear-algebra/onenormest.m, miscellaneous/perl.m,
-        plot/linkprop.m, plot/plot.m, plot/print.m, plot/stem3.m,
-        signal/diffpara.m, signal/stft.m, sparse/pcg.m, sparse/pcr.m,
-        specfun/primes.m, special-matrix/hadamard.m, special-matrix/hankel.m,
-        special-matrix/invhilb.m, special-matrix/pascal.m,
-        special-matrix/rosser.m, special-matrix/toeplitz.m,
-        special-matrix/vander.m, special-matrix/wilkinson.m,
-        testfun/speed.m: Spellcheck documentation
+	* audio/lin2mu.m, general/common_size.m, general/interp1.m, 
+	general/interp2.m, general/interpn.m, general/quadl.m,
+	geometry/dsearchn.m, geometry/voronoi.m, geometry/voronoin.m,
+	image/hsv2rgb.m, image/imread.m, image/rgb2hsv.m,
+	linear-algebra/commutation_matrix.m, linear-algebra/condest.m,
+	linear-algebra/expm.m, linear-algebra/krylov.m,
+	linear-algebra/onenormest.m, miscellaneous/perl.m,
+	plot/linkprop.m, plot/plot.m, plot/print.m, plot/stem3.m,
+	signal/diffpara.m, signal/stft.m, sparse/pcg.m, sparse/pcr.m,
+	specfun/primes.m, special-matrix/hadamard.m, special-matrix/hankel.m,
+	special-matrix/invhilb.m, special-matrix/pascal.m,
+	special-matrix/rosser.m, special-matrix/toeplitz.m,
+	special-matrix/vander.m, special-matrix/wilkinson.m,
+	testfun/speed.m: Spellcheck documentation
 
 2010-07-15  Jaroslav Hajek  <highegg@gmail.com>
 
 	* general/flipdim.m: Suuport flipping about trailing singleton
 	dimension.
 
 2010-07-14  Ben Abbott <bpabbott@mac.com>
 
@@ -209,29 +209,29 @@ 2010-07-07  David Bateman  <dbateman@fre
 
 2010-07-04  Soren Hauberg  <hauberg@gmail.com>
 
 	* polynomial/deconv.m: ensure that the orientation of the third
 	input to 'filter' matches the orientation of 'y'.
 
 2010-07-02  Rik <octave@nomad.inbox5.com>
 
-        * audio/wavread.m: Correctly handle non-word aligned data chunks.
-        (bug #30309).
+	* audio/wavread.m: Correctly handle non-word aligned data chunks.
+	(bug #30309).
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
-        * plot/stairs.m: Add additional calling forms to documentation.
+	* plot/stairs.m: Add additional calling forms to documentation.
 
 2010-06-24  John W. Eaton  <jwe@octave.org>
 
 	* plot/private/__plt__.m: Replace usage message with error.
 	* plot/loglog.m, plot/plot.m, plot/polar.m, plot/semilogx.m,
 	plot/semilogy.m: Call print_usage if no args are supplied.
-        Add additional calling forms to documentation.
+	Add additional calling forms to documentation.
 
 2010-06-24  Shai Ayal  <shaiay@users.sourceforge.net>
 
 	* plot/print.m: convert to wrapper for backend specific print functions
 	* plot/__gnuplot_print__.m: New gnuplot specific print function,
 	largely based on the former print.m
 	* plot/__fltk_print__.m: New fltk specific print function
 	* plot/__print_parse_opts__.m: New option parser for print
@@ -246,178 +246,178 @@ 2010-06-23  David Bateman  <dbateman@fre
 	background color.
 
 2010-06-17  Brad Froehle <brad.froehle@gmail.com>
 
 	* sparse/spy.m: Fix typo.
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
-        * set/union.m: Remove seealso reference to deprecated function.
+	* set/union.m: Remove seealso reference to deprecated function.
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
-        * @ftp/dir.m, @ftp/ftp.m, @ftp/mget.m, @ftp/mput.m, @ftp/rename.m,
-        deprecated/fstat.m, general/accumdim.m, general/interp1.m,
-        linear-algebra/ishermitian.m, linear-algebra/issymmetric.m,
-        miscellaneous/substruct.m, optimization/fminbnd.m,
-        optimization/fsolve.m, plot/bar.m, plot/daspect.m, plot/pbaspect.m,
-        plot/print.m, polynomial/mkpp.m, polynomial/ppval.m, set/union.m,
-        signal/unwrap.m, specfun/nthroot.m: Periodic grammar check of
-        documentation to ensure a common format.
+	* @ftp/dir.m, @ftp/ftp.m, @ftp/mget.m, @ftp/mput.m, @ftp/rename.m,
+	deprecated/fstat.m, general/accumdim.m, general/interp1.m,
+	linear-algebra/ishermitian.m, linear-algebra/issymmetric.m,
+	miscellaneous/substruct.m, optimization/fminbnd.m,
+	optimization/fsolve.m, plot/bar.m, plot/daspect.m, plot/pbaspect.m,
+	plot/print.m, polynomial/mkpp.m, polynomial/ppval.m, set/union.m,
+	signal/unwrap.m, specfun/nthroot.m: Periodic grammar check of
+	documentation to ensure a common format.
 
 2010-06-14  Rik <octave@nomad.inbox5.com>
 
-        * plot/image.m: Flip data, rather than y-axis, for images.  (bug #30033)
+	* plot/image.m: Flip data, rather than y-axis, for images.  (bug #30033)
 
 2010-06-12  Ben Abbott <bpabbott@mac.com>
 
 	* plot/scatter.m: New demo.
 	* plot/private/__scatter__.m: Properly set default value markersize
 	and marker type.
 
 2010-06-12  Rik <octave@nomad.inbox5.com>
 
-        * plot/scatter.m: Add new calling forms of function to documentation.
+	* plot/scatter.m: Add new calling forms of function to documentation.
 
 2010-06-10  Ben Abbott <bpabbott@mac.com>
 
 	* plot/private/__scatter__.m, plot/scatter.m: Honor Matlab color
 	settings.
 	* general/prepad.m: Fix typo 'dimm' -> 'dim'.
 
 2010-06-10  Jaroslav Hajek  <highegg@gmail.com>
 
 	* linear-algebra/vec.m: Remove (make built-in).
 	* linear-algebra/module.mk: Update.
 
 2010-06-09  Rik <octave@nomad.inbox5.com>
 
-        * general/flipdim.m, general/accumdim.m: Use > 1 test to find
-        first non-singleton dimension rather than != 1.
+	* general/flipdim.m, general/accumdim.m: Use > 1 test to find
+	first non-singleton dimension rather than != 1.
 
 2010-06-08  Rik <octave@nomad.inbox5.com>
 
-        * general/cumtrapz.m, general/postpad.m, general/prepad.m,
-        general/shift.m, general/trapz.m, signal/unwrap.m: Use common
-        method to find first non-singleton dimension.
+	* general/cumtrapz.m, general/postpad.m, general/prepad.m,
+	general/shift.m, general/trapz.m, signal/unwrap.m: Use common
+	method to find first non-singleton dimension.
 
 2010-06-08  Rik <octave@nomad.inbox5.com>
 
-        * general/rotdim.m: Modify function to use same variable names
-        as documentation.
+	* general/rotdim.m: Modify function to use same variable names
+	as documentation.
 
 2010-06-08  Rik <octave@nomad.inbox5.com>
 
-        * general/cart2pol.m, general/cart2sph.m, general/pol2cart.m,
-        general/sph2cart.m: Add option to operate on column matrix of
-        coordinates.
+	* general/cart2pol.m, general/cart2sph.m, general/pol2cart.m,
+	general/sph2cart.m: Add option to operate on column matrix of
+	coordinates.
 
 2010-06-08  Rik <octave@nomad.inbox5.com>
 
-        * general/arrayfun.m, general/cart2pol.m, general/cart2sph.m,
-        general/idivide.m, general/logspace.m, general/sph2cart.m,
-        general/structfun.m, image/imagesc.m, io/csvread.m, io/csvwrite.m,
-        linear-algebra/cond.m, linear-algebra/condest.m,
-        linear-algebra/isdefinite.m, linear-algebra/null.m,
-        linear-algebra/orth.m, miscellaneous/list_primes.m,
-        optimization/fminunc.m, optimization/fzero.m, path/matlabroot.m,
-        plot/axis.m, plot/plot.m, plot/semilogx.m, plot/semilogxerr.m,
-        plot/semilogy.m, plot/semilogyerr.m, plot/stem.m, polynomial/poly.m,
-        polynomial/polyder.m, polynomial/polyderiv.m, polynomial/polyfit.m,
-        polynomial/polyout.m, polynomial/polyval.m, set/ismember.m,
-        sparse/spfun.m, specfun/betaln.m, statistics/base/mean.m,
-        statistics/base/median.m, statistics/base/mode.m, statistics/base/ols.m,
-        statistics/base/ranks.m,
-        statistics/distributions/kolmogorov_smirnov_cdf.m,
-        statistics/distributions/wblpdf.m: Improve documentation strings
+	* general/arrayfun.m, general/cart2pol.m, general/cart2sph.m,
+	general/idivide.m, general/logspace.m, general/sph2cart.m,
+	general/structfun.m, image/imagesc.m, io/csvread.m, io/csvwrite.m,
+	linear-algebra/cond.m, linear-algebra/condest.m,
+	linear-algebra/isdefinite.m, linear-algebra/null.m,
+	linear-algebra/orth.m, miscellaneous/list_primes.m,
+	optimization/fminunc.m, optimization/fzero.m, path/matlabroot.m,
+	plot/axis.m, plot/plot.m, plot/semilogx.m, plot/semilogxerr.m,
+	plot/semilogy.m, plot/semilogyerr.m, plot/stem.m, polynomial/poly.m,
+	polynomial/polyder.m, polynomial/polyderiv.m, polynomial/polyfit.m,
+	polynomial/polyout.m, polynomial/polyval.m, set/ismember.m,
+	sparse/spfun.m, specfun/betaln.m, statistics/base/mean.m,
+	statistics/base/median.m, statistics/base/mode.m, statistics/base/ols.m,
+	statistics/base/ranks.m,
+	statistics/distributions/kolmogorov_smirnov_cdf.m,
+	statistics/distributions/wblpdf.m: Improve documentation strings
 
 2010-06-07  Jaroslav Hajek  <highegg@gmail.com>
 
 	* pkg/pkg.m: Support pkg list -forge.
 
 2010-06-07  Jaroslav Hajek  <highegg@gmail.com>
 
 	* pkg/get_forge_pkg.m: New function.
 	* pkg/module.mk: Add it here.
 	* pkg/pkg.m: Support -forge switch for auto-downloading OctaveForge
 	packages.
 
 2010-06-03  Alois Schlögl
 
-        * signal/arch_test.m, statistics/tests/bartlett_test.m,
-        statistics/tests/chisquare_test_homogeneity.m,
-        statistics/tests/chisquare_test_independence.m,
-        statistics/tests/manova.m, statistics/tests/mcnemar_test.m,
-        statistics/tests/run_test.m, statistics/distributions/chi2pdf.m:
-        Replace calls to obsolete chisquare_cdf with chi2cdf.
+	* signal/arch_test.m, statistics/tests/bartlett_test.m,
+	statistics/tests/chisquare_test_homogeneity.m,
+	statistics/tests/chisquare_test_independence.m,
+	statistics/tests/manova.m, statistics/tests/mcnemar_test.m,
+	statistics/tests/run_test.m, statistics/distributions/chi2pdf.m:
+	Replace calls to obsolete chisquare_cdf with chi2cdf.
 
 2010-06-03  Rik <octave@nomad.inbox5.com>
 
 	* plot/private/__plt__.m: Add newline to usage message.
 
 2010-06-02  Rik <octave@nomad.inbox5.com>
 
 	* optimization/sqp.m: Overhaul sqp code.
-        Guarantee equivalent results if sqp called with or wihout bounds
-        (bug #29989).  Simplify input option handling and add %tests
-        to check validation code.  Rewrite documentation string.
+	Guarantee equivalent results if sqp called with or wihout bounds
+	(bug #29989).  Simplify input option handling and add %tests
+	to check validation code.  Rewrite documentation string.
 
 2010-06-01  Rik <octave@nomad.inbox5.com>
 
 	* optimization/fminbnd.m: Remove unused persistent variable.
 
 2010-06-01  Rik <octave@nomad.inbox5.com>
 
 	* plot/private/__contour__.m: Correctly size X and Y when only Z
-        specified (bug #30002).
+	specified (bug #30002).
 
 2010-05-31  Rik <octave@nomad.inbox5.com>
 
 	* specfun/legendre.m: Orient row vector correctly (bug #29997).
-        Add input validation for negative values and %tests to check
-        validation routines.
+	Add input validation for negative values and %tests to check
+	validation routines.
 
 2010-05-30  Rik <octave@nomad.inbox5.com>
 
-        * sparse/svds.m: Overhaul code.
-        Return smallest singular values if sigma == 0 (Bug #29721).
-        Avoid calculating U and V matrices unless requested.
-        Correctly handle zero matrix input.
-        Improve documentation string.
+	* sparse/svds.m: Overhaul code.
+	Return smallest singular values if sigma == 0 (Bug #29721).
+	Avoid calculating U and V matrices unless requested.
+	Correctly handle zero matrix input.
+	Improve documentation string.
 
 2010-05-26  Rik <octave@nomad.inbox5.com>
 
-        * statistics/base/histc.m, statistics/base/iqr.m,
-        statistics/base/kurtosis.m, statistics/base/mode.m,
-        statistics/base/ranks.m, statistics/base/run_count.m,
-        statistics/base/skewness.m, statistics/base/statistics.m,
-        statistics/base/studentize.m: Improve validation of input arguments.
+	* statistics/base/histc.m, statistics/base/iqr.m,
+	statistics/base/kurtosis.m, statistics/base/mode.m,
+	statistics/base/ranks.m, statistics/base/run_count.m,
+	statistics/base/skewness.m, statistics/base/statistics.m,
+	statistics/base/studentize.m: Improve validation of input arguments.
 
 2010-05-26  Jaroslav Hajek  <highegg@gmail.com>
 
 	* specfun/isprime.m: Fix and further optimize.
 
 2010-05-26  Rik <octave@nomad.inbox5.com>
 
-        * plot/legend.m: Improve documentation string.
+	* plot/legend.m: Improve documentation string.
 
 2010-05-26  Rik <octave@nomad.inbox5.com>
 
-        * sparse/svds.m: Check struct input arguments.  Overhaul documentation.
+	* sparse/svds.m: Check struct input arguments.  Overhaul documentation.
 
 2010-05-26  Ben Abbott <bpabbott@mac.com>
 
 	* plot/legend.m: Key labels for specified objects.
 	* plot/ancestor.m: Support vectors of handles.
 
 2010-05-26  Rik <octave@nomad.inbox5.com>
 
-        * sparse/svds.m: Initialize ARPACK starting vector in svds test code.
-        Bug #29721.
+	* sparse/svds.m: Initialize ARPACK starting vector in svds test code.
+	Bug #29721.
 
 2010-05-25  Ben Abbott <bpabbott@mac.com>
 
 	* polynomial/spline.m: Ignore NaNs within input vectors.
 
 2010-05-25  Jaroslav Hajek  <highegg@gmail.com>
 
 	* specfun/primes.m: Use logical masks rather than numeric.
@@ -428,90 +428,90 @@ 2010-05-25  Jaroslav Hajek  <highegg@gma
 	* miscellaneous/unimplemented.m: Don't mention onCleanup (supported).
 
 2010-05-25  Jaroslav Hajek  <highegg@gmail.com>
 
 	* statistics/base/median.m: Ensure NaNs where needed. Simplify first
 	non-singleton dimension computation.
 
 2010-05-24  Rik <octave@nomad.inbox5.com>
-        * general/private/__isequal__.m, image/image.m,
-        statistics/base/ranks.m: Replace calls to deprecated functions.
+	* general/private/__isequal__.m, image/image.m,
+	statistics/base/ranks.m: Replace calls to deprecated functions.
 
 2010-05-23  Rik <octave@nomad.inbox5.com>
 
-        * image/imshow.m: Scale image point color according to colormap.
-        Bug #29926.
+	* image/imshow.m: Scale image point color according to colormap.
+	Bug #29926.
 
 2010-05-16  Rik <octave@nomad.inbox5.com>
 
-        * plot/semilogx.m, plot/semilogy.m: Plot minor ticks for semilog plots.
+	* plot/semilogx.m, plot/semilogy.m: Plot minor ticks for semilog plots.
 
 2010-05-16  Rik <octave@nomad.inbox5.com>
 
-        * audio/loadaudio.m, audio/playaudio.m, audio/saveaudio.m,
-        audio/wavread.m, general/bicubic.m, general/bitcmp.m, general/bitget.m,
-        general/bitset.m, general/cplxpair.m, general/del2.m,
-        general/gradient.m, general/interp2.m, general/interp3.m,
-        general/interpn.m, general/postpad.m, general/prepad.m,
-        general/private/__splinen__.m, image/image_viewer.m, image/imagesc.m,
-        image/saveimage.m, linear-algebra/expm.m, linear-algebra/krylov.m,
-        miscellaneous/edit.m, miscellaneous/private/__xzip__.m,
-        miscellaneous/what.m, optimization/glpk.m, optimization/optimset.m,
-        optimization/pqpnonneg.m, path/pathdef.m, plot/__go_draw_axes__.m,
-        plot/__marching_cube__.m, plot/gnuplot_drawnow.m, plot/ishold.m,
-        plot/isocolors.m, plot/isonormals.m, plot/print.m,
-        plot/private/__add_datasource__.m, plot/private/__color_str_rgb__.m,
-        plot/private/__errplot__.m, plot/private/__interp_cube__.m,
-        plot/shading.m, plot/specular.m, polynomial/deconv.m,
-        polynomial/polyint.m, polynomial/ppint.m, set/powerset.m,
-        signal/arch_fit.m, signal/arch_test.m, signal/durbinlevinson.m,
-        special-matrix/hadamard.m, statistics/distributions/unidrnd.m,
-        statistics/distributions/unifinv.m,
-        statistics/models/logistic_regression.m, strings/strjust.m,
-        testfun/rundemos.m, testfun/runtests.m, testfun/test.m, time/datenum.m:
-        Print name of function in error() string messages.
+	* audio/loadaudio.m, audio/playaudio.m, audio/saveaudio.m,
+	audio/wavread.m, general/bicubic.m, general/bitcmp.m, general/bitget.m,
+	general/bitset.m, general/cplxpair.m, general/del2.m,
+	general/gradient.m, general/interp2.m, general/interp3.m,
+	general/interpn.m, general/postpad.m, general/prepad.m,
+	general/private/__splinen__.m, image/image_viewer.m, image/imagesc.m,
+	image/saveimage.m, linear-algebra/expm.m, linear-algebra/krylov.m,
+	miscellaneous/edit.m, miscellaneous/private/__xzip__.m,
+	miscellaneous/what.m, optimization/glpk.m, optimization/optimset.m,
+	optimization/pqpnonneg.m, path/pathdef.m, plot/__go_draw_axes__.m,
+	plot/__marching_cube__.m, plot/gnuplot_drawnow.m, plot/ishold.m,
+	plot/isocolors.m, plot/isonormals.m, plot/print.m,
+	plot/private/__add_datasource__.m, plot/private/__color_str_rgb__.m,
+	plot/private/__errplot__.m, plot/private/__interp_cube__.m,
+	plot/shading.m, plot/specular.m, polynomial/deconv.m,
+	polynomial/polyint.m, polynomial/ppint.m, set/powerset.m,
+	signal/arch_fit.m, signal/arch_test.m, signal/durbinlevinson.m,
+	special-matrix/hadamard.m, statistics/distributions/unidrnd.m,
+	statistics/distributions/unifinv.m,
+	statistics/models/logistic_regression.m, strings/strjust.m,
+	testfun/rundemos.m, testfun/runtests.m, testfun/test.m, time/datenum.m:
+	Print name of function in error() string messages.
 
 2010-05-16  Rik <octave@nomad.inbox5.com>
 
-        * scripts/plot/contourc.m, scripts/plot/private/__contour__.m,
-          scripts/plot/surface.m, scripts/plot/surfnorm.m: Check input
-          arguments for size and type (bug #29861).
+	* scripts/plot/contourc.m, scripts/plot/private/__contour__.m,
+	  scripts/plot/surface.m, scripts/plot/surfnorm.m: Check input
+	  arguments for size and type (bug #29861).
 
 2010-05-13  John W. Eaton  <jwe@octave.org>
 
 	* help/help.m: Avoid unnecessary use of varargout.
 
 2010-05-13   Qianqian Fang <fangq>
 
-        * miscellaneous/tempdir.m: Add filesep to end of tempdir (bug #29864).
+	* miscellaneous/tempdir.m: Add filesep to end of tempdir (bug #29864).
 
 2010-05-11  Jaroslav Hajek  <highegg@gmail.com>
 
 	* io/textread.m: Fix typo.
 
 2010-05-08  David Gesswein <djg@pdp8online.com>
 
-        * io/dlmwrite.m: Correctly set output precision (#29870).
+	* io/dlmwrite.m: Correctly set output precision (#29870).
 
 2010-05-12  Jordi Gutiérrez Hermoso <jordigh@gmail.com>
 
 	* help/help.m: Allow docstring to be returned as output, and
 	correct docstring.
 
 2010-05-08  Rik <octave@nomad.inbox5.com>
 
-        * testfun/test.m: Don't abort testing when skipping a testif block
+	* testfun/test.m: Don't abort testing when skipping a testif block
 
 2010-05-05  Rik <octave@nomad.inbox5.com>
 
-        * statistics/tests/cor_test.m, statistics/tests/t_test.m,
-          statistics/tests/t_test_2.m, statistics/tests/t_test_regression.m,
-          statistics/tests/welch_test.m: Replace deprecated function calls.
-          Patch provided by Alois Schlögl.
+	* statistics/tests/cor_test.m, statistics/tests/t_test.m,
+	  statistics/tests/t_test_2.m, statistics/tests/t_test_regression.m,
+	  statistics/tests/welch_test.m: Replace deprecated function calls.
+	  Patch provided by Alois Schlögl.
 
 2010-05-02  Rik <octave@nomad.inbox5.com>
 
 	* linear-algebra/cond.m: Add TeX equation to documentation
 
 2010-05-02  Peter van den Biggelaar <peter@biglr.net>
 
 	* plot/close.m: No error on "close([])".
diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,27 +1,28 @@
 2010-07-18  Kai Habel <kai.habel@gmx.de>
-        * DLD-FUNCTIONS/fltk_backend.cc: Add mode for mouse rotation.
-        (view2status): new function
-        * src/graphics.cc: Remove limitations for zooming and translation.
-        (axes::properties::rotate_view): new function
-        
+
+	* DLD-FUNCTIONS/fltk_backend.cc: Add mode for mouse rotation.
+	(view2status): new function
+	* src/graphics.cc: Remove limitations for zooming and translation.
+	(axes::properties::rotate_view): new function
+	
 2010-07-17  Rik <octave@nomad.inbox5.com>
 
-        * DLD-FUNCTIONS/balance.cc, DLD-FUNCTIONS/ccolamd.cc,
-          DLD-FUNCTIONS/cellfun.cc, DLD-FUNCTIONS/dasrt.cc,
-          DLD-FUNCTIONS/dmperm.cc, DLD-FUNCTIONS/eigs.cc, DLD-FUNCTIONS/qz.cc,
-          DLD-FUNCTIONS/rand.cc, DLD-FUNCTIONS/sparse.cc, 
-          DLD-FUNCTIONS/sqrtm.cc, DLD-FUNCTIONS/symrcm.cc, dirfns.cc,
-          file-io.cc, syscalls.cc: Spellcheck documentation
+	* DLD-FUNCTIONS/balance.cc, DLD-FUNCTIONS/ccolamd.cc,
+	  DLD-FUNCTIONS/cellfun.cc, DLD-FUNCTIONS/dasrt.cc,
+	  DLD-FUNCTIONS/dmperm.cc, DLD-FUNCTIONS/eigs.cc, DLD-FUNCTIONS/qz.cc,
+	  DLD-FUNCTIONS/rand.cc, DLD-FUNCTIONS/sparse.cc, 
+	  DLD-FUNCTIONS/sqrtm.cc, DLD-FUNCTIONS/symrcm.cc, dirfns.cc,
+	  file-io.cc, syscalls.cc: Spellcheck documentation
 
 2010-07-08  Rik <octave@nomad.inbox5.com>
 
 	* DLD_FUNCTIONS/rand.cc: Correct typo in rande documentation.
-        (bug #30446).
+	(bug #30446).
 
 2010-07-13  Jaroslav Hajek  <highegg@gmail.com>
 
 	* DLD-FUNCTIONS/cellfun.cc (mat2cell_mismatch, prepare_idx,
 	do_mat2cell_2d, do_mat2cell_nd, do_mat2cell): New helper funcs.
 	(Fmat2cell): Use them here.
 
 2010-07-13  Jaroslav Hajek  <highegg@gmail.com>
@@ -43,33 +44,33 @@ 2010-07-12  Jaroslav Hajek  <highegg@gma
 2010-07-08  David Bateman  <dbateman@free.fr>
 
 	* DLD-FUNCTIONS/__magick_read__,cc (F__magick_read__): Add the syntax
 	__magick_read__ ("file", "frames", "all").
 
 2010-07-08  Rik <octave@nomad.inbox5.com>
 
 	* DLD_FUNCTIONS/eigs.cc: Initialize random number generator in test
-        blocks to get reproducible results.
+	blocks to get reproducible results.
 
 2010-07-08  Jaroslav Hajek  <highegg@gmail.com>
 
 	* ov-cell.cc (Fstruct2cell): Use octave_map. Optimize.
 
 2010-07-07  Rik <octave@nomad.inbox5.com>
 
 	* pr-output.cc (calc_scale_exp): Avoid use of % operator on negative
-        operands for portability between C++ compilers.
-        (engineering_exponent): Avoid use of % operator.  Keep calculation of
-        exponent within pr_engineering_float class.  (encapsulation).
-        (num_digits): Rename from calc_digits.  Return true number of digits =
-        1 + calc_digits.
-        (calc_scale_exp): Rename from maybe_mod3 for readability.
-        (operator << (... pr_engineering_float pef): Use pef abbreviation for
-        pr_engineering_float.
+	operands for portability between C++ compilers.
+	(engineering_exponent): Avoid use of % operator.  Keep calculation of
+	exponent within pr_engineering_float class.  (encapsulation).
+	(num_digits): Rename from calc_digits.  Return true number of digits =
+	1 + calc_digits.
+	(calc_scale_exp): Rename from maybe_mod3 for readability.
+	(operator << (... pr_engineering_float pef): Use pef abbreviation for
+	pr_engineering_float.
 
 2010-07-07  David Bateman  <dbateman@free.fr>
 
 	* graphics.h.in (axes): Add hidden property __hold_all.
 
 2010-07-07  Jaroslav Hajek  <highegg@gmail.com>
 
 	* ov-class.h (octave_class::map): Turn to octave_map.
@@ -204,17 +205,17 @@ 2010-06-25  John W. Eaton  <jwe@octave.o
 
 	* graphics.cc (reset_default_properties): New static function.
 	(root_figure::reset_default_properties,
 	figure::reset_default_properties, axes::reset_default_properties):
 	Use it to avoid duplicated code.
 
 2010-06-24  Rik <octave@nomad.inbox5.com>
 
-        * octave.cc: Add [FILE] to octave usage string (bug #30258).
+	* octave.cc: Add [FILE] to octave usage string (bug #30258).
 
 2010-06-24  Jaroslav Hajek  <highegg@gmail.com>
 
 	* oct-map.h (octave_map, octave_scalar_map): New classes.
 	* oct-map.cc (octave_map, octave_scalar_map): Implement them.
 	(Octave_map::Octave_map (const octave_map&)): New ctor.
 	* ov-struct.h (octave_struct): Use octave_map for storage.
 	(octave_scalar_struct): New class.
@@ -241,17 +242,17 @@ 2010-06-23  David Bateman  <dbateman@fre
 	void figure::reset_default_properties (void),
 	void figure::reset_default_properties (void)): Declarations.
 	(std::string color_property::values_as_string (void) const,
 	Cell color_property::values_as_cell (void) const): New functions.
 
 2010-06-21  Rik <octave@nomad.inbox5.com>
 
 	* DLD-FUNCTIONS/dlmread.cc: Fix spreadsheet style range option.
-        (bug #30183).
+	(bug #30183).
 
 2010-06-21  David Bateman  <dbateman@free.fr>
 
 	* graphics.cc (std::string radio_values:values_as_string (void)
 	const, Cell radio_values:values_as_cell (void) const): New functions.
 	(void graphics_object::set (const octave_value_list&)): Throw error
 	with no arguments.
 	(std::string base_graphics_object::values_as_string (void),
@@ -290,22 +291,22 @@ 2010-06-18  Jaroslav Hajek  <highegg@gma
 	(tm_row_const::do_init_element): Simplify using dim_vector::hvcat.
 	(tm_const::init): Ditto.
 	(single_type_concat): Special-case empty results. Skip or use 0x0 for
 	empty arrays otherwise.
 	(tree_matrix::rvalue1): Skip empty arrays in the fallback branch.
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
-        * DLD-FUNCTIONS/cellfun.cc, DLD-FUNCTIONS/dot.cc,
-        DLD-FUNCTIONS/gammainc.cc, DLD-FUNCTIONS/lookup.cc,
-        DLD-FUNCTIONS/lu.cc, DLD-FUNCTIONS/luinc.cc, DLD-FUNCTIONS/onCleanup.cc,
-        DLD-FUNCTIONS/sparse.cc, DLD-FUNCTIONS/spparms.cc, DLD-FUNCTIONS/svd.cc,
-        data.cc, dirfns.cc, graphics.cc, mappers.cc, ov-cell.cc, ov-range.cc:
-        Periodic grammar check of documentation to ensure a common format.
+	* DLD-FUNCTIONS/cellfun.cc, DLD-FUNCTIONS/dot.cc,
+	DLD-FUNCTIONS/gammainc.cc, DLD-FUNCTIONS/lookup.cc,
+	DLD-FUNCTIONS/lu.cc, DLD-FUNCTIONS/luinc.cc, DLD-FUNCTIONS/onCleanup.cc,
+	DLD-FUNCTIONS/sparse.cc, DLD-FUNCTIONS/spparms.cc, DLD-FUNCTIONS/svd.cc,
+	data.cc, dirfns.cc, graphics.cc, mappers.cc, ov-cell.cc, ov-range.cc:
+	Periodic grammar check of documentation to ensure a common format.
 
 2010-06-11  David Bateman  <dbateman@free.fr>
 
 	* DLD-FUNCTIONS/rand.cc : Add "reset" flag to reset the random
 	generators from the entropy pool.
 
 	* genprops.awk : Don't special case the color_property in the
 	factory default methods.
@@ -316,17 +317,17 @@ 2010-06-16  Jaroslav Hajek  <highegg@gma
 
 2010-06-15  Rik <octave@nomad.inbox5.com>
 
 	* pr-output.cc: Second try at removing extra newlines from NDArrays
 
 2010-06-14  Rik <octave@nomad.inbox5.com>
 
 	* pr-output.cc: Remove extra newlines when displaying multi-dimensional
-        arrays (bug #30029).
+	arrays (bug #30029).
 
 2010-06-14  Jaroslav Hajek  <highegg@gmail.com>
 
 	* data.cc (Fsort, Fnth_element): Fix behavior when trailing singleton
 	dim is specified.
 
 2010-06-12  Ben Abbott <bpabbott@mac.com>
 
@@ -352,20 +353,20 @@ 2010-06-09  Jaroslav Hajek  <highegg@gma
 	arguments. Simplify computing unknown dimension.
 
 2010-06-09  David Grundberg  <individ@acc.umu.se>
 
 	* ov-struct.cc (Fisfield): Accept cell array as name argument.
 
 2010-06-08  Rik <octave@nomad.inbox5.com>
 
-        * DLD-FUNCTIONS/bsxfun.cc, DLD-FUNCTIONS/cellfun.cc,
-        DLD-FUNCTIONS/det.cc, DLD-FUNCTIONS/lookup.cc, DLD-FUNCTIONS/lu.cc,
-        DLD-FUNCTIONS/rand.cc, DLD-FUNCTIONS/svd.cc, DLD-FUNCTIONS/tril.cc,
-        data.cc, ov.cc: Improve documentation string.
+	* DLD-FUNCTIONS/bsxfun.cc, DLD-FUNCTIONS/cellfun.cc,
+	DLD-FUNCTIONS/det.cc, DLD-FUNCTIONS/lookup.cc, DLD-FUNCTIONS/lu.cc,
+	DLD-FUNCTIONS/rand.cc, DLD-FUNCTIONS/svd.cc, DLD-FUNCTIONS/tril.cc,
+	data.cc, ov.cc: Improve documentation string.
 
 2010-06-06  Michael Goffioul  <michael.goffioul@gmail.com>
 
 	* data.cc (Frem, Fmod): Specify template arguments of binmap
 	explicitely.
 
 2010-06-02  Jaroslav Hajek  <highegg@gmail.com>
 
@@ -396,17 +397,17 @@ 2010-05-28  Jaroslav Hajek  <highegg@gma
 	octave_base_matrix<Cell>::fast_elem_insert): New specializations.
 	* DLD-FUNCTIONS/cellfun.cc (scalar_col_helper, scalar_col_helper_def,
 	scalar_col_helper_nda, make_col_helper, can_extract): Remove.
 	(Fcellfun): Use the new fast_elem_insert method.
 
 2010-05-10  Rik <octave@nomad.inbox5.com>
 
 	* DLD-FUNCTIONS/eigs.cc: Improve documentation string.
-        Use ~ notation instead of dummy variables for unused return arguments.
+	Use ~ notation instead of dummy variables for unused return arguments.
 
 2010-05-27  Jaroslav Hajek  <highegg@gmail.com>
 
 	* ov-base-mat.h (octave_base_matrix::matrix_ref (void) const): New
 	overload.
 
 2010-05-26  Jaroslav Hajek  <highegg@gmail.com>
 
@@ -523,17 +524,17 @@ 2010-05-11  Jaroslav Hajek  <highegg@gma
 2010-05-07  Jaroslav Hajek  <highegg@gmail.com>
 
 	* DLD-FUNCTIONS/sqrtm.cc (do_sqrtm): Mark the result as triangular
 	where applicable.
 
 2010-05-10  Rik <octave@nomad.inbox5.com>
 
 	* DLD-FUNCTIONS/qr.cc: Split qrdelete test into high precision test
-        when QRUPDATE is present and low precision when it is not.
+	when QRUPDATE is present and low precision when it is not.
 
 2010-05-10  Jaroslav Hajek  <highegg@gmail.com>
 
 	* DLD-FUNCTIONS/schur.cc (Fschur): Mark triangular matrices on output.
 
 2010-05-10  Jaroslav Hajek  <highegg@gmail.com>
 
 	* ov-range.cc (octave_range::index_vector): Optimize order of
@@ -607,17 +608,17 @@ 2010-05-03  Jaroslav Hajek  <highegg@gma
 
 	* DLD-FUNCTIONS/svd.cc (driver): New static var.
 	(Fsvd): Use it to select a driver.
 	(Fsvd_driver): New DEFUN.
 
 2010-05-02  Rik <octave@nomad.inbox5.com>
 
 	* ov-cell.cc: Wrap documentation line to prevent overfull hbox
-        LaTex warning
+	LaTex warning
 
 2010-05-01  Ben Abbott <bpabbott@mac.com>
 
 	* graphics.cc, graphics.h.in: Remove line object properties ldata,
 	udata, xldata, xudata.
 
 2010-04-30  Shai Ayal  <shaiay@users.sourceforge.net>
 
@@ -649,17 +650,17 @@ 2010-04-27  John W. Eaton  <jwe@octave.o
 	* oct-parse.yy (source_file): Limit recursive calls on a
 	per-file basis.
 	Addresses bug #29491.
 
 2010-04-25  David Bateman  <dbateman@free.fr>
 
 	* oct-parse.yy (octave_function * load_fcn_from_file
 	(const std::string&, const std::string&, const std::string&)
-        (const std::string&, bool)): When loading a mex-file attempt
+	(const std::string&, bool)): When loading a mex-file attempt
 	to load the m-file first to get the help string.
 
 2010-04-24  Ben Abbott <bpabbott@mac.com>
 
 	* graphics.h.in: Have figure color property support a radio value,
 	'none'.
 
 2010-04-24  David Bateman  <dbateman@free.fr>
@@ -2142,17 +2143,17 @@ 2010-01-17  Jaroslav Hajek  <highegg@gma
 
 2010-01-17  Thorsten Meyer  <thorsten.meyier@gmx.de>
 
 	* ov-cell.cc (Fstruct2cell): Add example, add test.
 	* DLD-FUNCTIONS/cellfun.cc (Fcellfun, Fnum2cell), ov-struct.cc
 	(Fisstruct): Fix docstring.
 	* DLD-FUNCTIONS/cellfun.cc (Fcellslices, Fisfield, Ffieldnames,
 	Frmfield): Add test.
-        * ov-struct.cc (Fcell2struct): Fix docstring, add test.
+	* ov-struct.cc (Fcell2struct): Fix docstring, add test.
 
 2010-01-16  David Bateman  <dbateman@free.fr>
 
 	* graphics.h.in (line::properties, surface::properties,
 	patch::properties): Allow the use of a "@" marker.
 	(axes::properties): Add markerorder properties to set the
 	order of the marker styles for the "@" marker.
 
